pkg/cri/server/container_create.go

```diff
@@ -332,11 +342,16 @@ func (c *criService) volumeMounts(containerRootDir string, criMounts []*runtime.
 		}
 		volumeID := util.GenerateID()
 		src := filepath.Join(containerRootDir, "volumes", volumeID)
+		if !filepath.IsAbs(dst) && goruntime.GOOS != "windows" {
+			dst = filepath.Join("/", dst)
```

Comment:
please consider to use `filepath.Clean`

Reply:
Marking as solved to unify the discussion in one thread, I can change it in all the relevant parts afterwards :)